qvisor/src/vmspace/mod.rs

```diff
@@ -159,6 +164,10 @@ impl VMSpace {
             vcpus: Vec::new(),
             haveMembarrierGlobal: haveMembarrierGlobal,
             haveMembarrierPrivateExpedited: haveMembarrierPrivateExpedited,
+            #[cfg(feature = "cc")]
```

Comment:
Actually previously these members were not stored in VMSpace because they are used directly later in the same function as parameters of initSharespace, while now cc initiates sharespace later, so they are stored here. And if you think it is better to store them whether it is cc, it is an alternative.

Reply:
Perhaps just store them here for non-cc as well, I don't see a performance loss to store some additional data here.